### PR TITLE
Don't reject WMTS sources by default

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

File: src/core.js [hunk 1]
```diff
@@ -368,11 +367,6 @@ olcs.core.tileLayerToImageryLayer = function(olLayer, viewProj) {
   let provider = null;
 
   const source = olLayer.getSource();
-  // handle special cases before the general synchronization
-  if (source instanceof ol.source.WMTS) {
```
Comment (gberaudo) [inline src/core.js]: Please modify the changelog to state we now handle WMTS sources, expecting the tilegrid is the same.

**Comment by fredj** [comment]
@gberaudo changelog updated

**Comment by zl142900** [comment]
Does ol-cesium support WMTS now?Why didn't I achieve it?

**Comment by gberaudo** [comment]
@zl142900, there is no example so I am not sure if everything is in OL-Cesium or not.
The best approach is to run your code step by step and see if there is an early exit somewhere.

**Comment by zl142900** [comment]
At present, my problem is that the WMTS can only be loaded in two dimensions. If you switch to three dimensions, the WMTS cannot be loaded

Do I need to set  in core.js
 if (source instanceof olSourceWMTS) {
      //What should I write
 }

**Pull request merged**